components/navigationBar/index.html

```diff
@@ -1,12 +1,12 @@
 <div>
-    <button v-if="back === true" class="back nav-btn" v-on="click: goBack">&lt;</button>
-    <a v-if="typeof back === 'string'" class="back nav-btn" href="{{back}}">&lt;</a>
+    <button v-if="back === true" class="back nav-btn" v-on="click: goBack"><span class="icon ion-android-arrow-back"></span></button>
+    <a v-if="typeof back === 'string'" class="back nav-btn" href="{{back}}"><span class="icon ion-android-arrow-back"></span></a>
 
     <button v-if="cancel === true" v-on="click: goBack" class="nav-btn">{{'Cancel' | i18n}}</button>
     <a v-if="typeof cancel === 'string'" href="{{cancel}}" class="nav-btn">{{'Cancel' | i18n}}</a>
 </div>
 
-<h1>{{title | i18n}}</h1>
+<h1>{{app.name}}</h1>
```

Comment:
This is used for all views, so changing this will affect them too -- instead, what you want to do in share/index.js is set the `title` property around here https://github.com/mozilla/webmaker-app/blob/master/views/share/index.js#L80.

So like
```
self.$data.title = app.name
```

Is it weird that child components like navigationBar automatically inherit their parent's `$data` properties? Definitely. Is it the way vue.js 10.6 works? Yes.